tests/test_infer/test_infer.py

```diff
@@ -107,6 +107,14 @@ def test_init(self):
                 'https://download.openmmlab.com/mmdetection/v2.0/resnest/faster_rcnn_s50_fpn_syncbn-backbone%2Bhead_mstrain-range_1x_coco/faster_rcnn_s50_fpn_syncbn-backbone%2Bhead_mstrain-range_1x_coco_20200926_125502-20289c16.pth',  # noqa: E501
             )
 
+        if is_installed('mmyolo'):
```

Comment:
Should it install mmyolo in CI?

Reply:
This is only for local test, otherwise, it will influence the statistics of the frequency of downloading the model weight.